angular/src/app/path/path.component.html

```diff
@@ -112,7 +112,7 @@
     <h3>{{ 'select_grade' | translate }}: {{ selectedInternship.name }}</h3>
     <div class="d-flex justify-content-center align-items-center">
       <label for="grade" class="me-2">{{ 'grade' | translate }}:</label>
-      <input type="number" id="grade" [(value)]="selectedGrade" min="100" max="125" class="form-control w-auto">
```

Comment:
Hey, did you delete the "id" attribute by accident?

Reply:
yep, I created a new commit where I add it back